public/pages/AnomalyCharts/containers/AnomalyHeatmapChart.tsx

```diff
@@ -109,28 +124,57 @@ export const AnomalyHeatmapChart = React.memo(
       });
 
       return [
-        COMBINED_OPTIONS,
+        getCombindeOptions(
+          COMBINED_OPTIONS,
+          props.heatmapDisplayOption?.entityOption
+        ),
         {
           label: 'Individual entities',
-          options: individualEntityOptions,
+          options: individualEntityOptions.reverse(),
         },
       ];
     };
 
+    const getCombindeOptions = (
+      existingOptions: any,
+      selectedCombinedOption: any | undefined
+    ) => {
+      if (!selectedCombinedOption) {
+        return existingOptions;
+      }
+      return {
+        label: existingOptions.label,
+        options: uniq([selectedCombinedOption, ...existingOptions.options]),
+      };
+    };
+
     const [originalHeatmapData, setOriginalHeatmapData] = useState(
-      getAnomaliesHeatmapData(
-        props.anomalies,
-        props.dateRange,
-        AnomalyHeatmapSortType.SEVERITY,
-        COMBINED_OPTIONS.options[0].value
-      )
+      props.showAlerts
```

Comment:
Can we add another prop to determine if we are using actual vs. sample results, rather than reusing the `showAlerts` one? One reason for this is that those may not always coincide with each other. For example, with historical detector results, `props.showAlerts` is `false` since alerts don't apply, but it is also not sample results.

I'm reusing the `AnomalyHistory` component, and have actually added a prop to handle this (as well as for all of the necessary children components) [here](https://github.com/ohltyler/anomaly-detection-kibana-plugin/commit/33b4c0c73364e6ed86dcfa4102032134d6d7fb49). I'm wondering if you can maybe do something similar here. Will help with possible merge conflicts down the road as well. Let me know what you think.

Reply:
Yeah, I was thinking about the same thing. Sure. I can add another prop for this case, something like `isUsingSampleAnomaly`

Reply:
Got it, sounds good.